marlin/Cargo.toml

```diff
@@ -73,15 +73,15 @@ default-features = false
 version = "1.3"
 
 [dependencies.blake2]
-version = "0.9"
+version = "0.10"
 default-features = false
 
 [dependencies.derivative]
 version = "2"
 features = [ "use_core" ]
 
 [dependencies.digest]
-version = "0.9"
+version = "0.10"
 
```

Comment:
Can you move `blake2` to a `dev-dep` (and check if `digest` can be moved down as well)?

Reply:
`digest` still needs to be a `dep` as it is used in the `FiatShamirChaCha` impl.